src/main/java/qna/domain/Question.java

```diff
@@ -39,12 +50,53 @@ public Question writeBy(User writer) {
         return this;
     }
 
+    public DeleteHistories delete(User loginUser) throws CannotDeleteException {
+        validationDeleteQuestionRequestUser(loginUser);
+        validationDeleteAnswersRequestUser(loginUser);
+
+        setDeleted(true);
+
+        return generateDeleteHistories();
+    }
+
     public boolean isOwner(User writer) {
         return this.writer.equals(writer);
     }
 
     public void addAnswer(Answer answer) {
         answer.toQuestion(this);
+        this.answers.add(answer);
+    }
+
+    public boolean isDeletedStatusAnswers() {
+        for (Answer answer : answers) {
+            if (!answer.isDeleted()) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
`indent` 가 두단계로 되어 있어서 조금 아쉬운 부분 같습니다. 😅
다음과 같이 stream 을 활용해보시면 명확하고 간결하게 변경할 수 있을 것 같은데 어떠신가요?
알고 계실 수도 있지만 참고하시면 좋을 것 같아 이외에도 스트림의 다양한 기능들에 대한 자료 남겨드리도록 하겠습니다!
[스트림 활용](https://velog.io/@kanamycine/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94%EC%9D%B8-%EC%95%A1%EC%85%98-chp-5.-%EC%8A%A4%ED%8A%B8%EB%A6%BC-%ED%99%9C%EC%9A%A9-1#03-%EA%B2%80%EC%83%89%EA%B3%BC-%EB%A7%A4%EC%B9%AD)
```suggestion
        return answers.stream()
                .allMatch(Answer::isDeleted);
```

Reply:
아닛! 전체적으로 보면서 indent 두단계인걸 확인했는데 이렇게 반영이 안된부분이 있다니 ㅠ_ㅠ
좀더 꼼꼼히 살펴봐야겠어요!